docs/node_scenarios.md

```diff
@@ -12,6 +12,7 @@ The following node chaos scenarios are supported:
 8. **restart_kubelet_scenario**: Scenario to restart the kubelet of the node instance.
 9. **node_crash_scenario**: Scenario to crash the node instance.
 10. **stop_start_helper_node_scenario**: Scenario to stop and start the helper node and check service status.
+11. **node_disk_detach_attach_scenario**: Scenario to detach node disk for specified duration.
```

Comment:
Now that you have all the scaffolding working, I'd add `node_disk_detach_scenario` and `node_disk_attach_scenario` as "base" operations and would remove the `duration` for this scenario.

Reply:
Withdraw the request to remove the `duration` parameter. Would you mind adding a mention in the `node_stop_start_scenario` description that it starts after a specified duration as well?